pkg/common/resources/resources.go

```diff
@@ -909,7 +909,7 @@ func CalculateAbsUsedCapacity(capacity, used *Resource) *Resource {
 	}
 	if missingResources.Len() != 0 {
 		log.Logger().Debug("Absolute usage result is missing resource information",
-			zap.String("missing resource(s)", missingResources.String()))
+			zap.Stringer("missing resource(s)", &missingResources))
```

Comment:
Can you check the effect of & ?

Reply:
Thank you @manirajv06 for review, I have checked the code here:
`
// String returns the accumulated string.
func (b *Builder) String() string {
	return *(*string)(unsafe.Pointer(&b.buf))
}
`

Reply:
A much cleaner solution would be to change the init of the variable to:
```
missingResources := &strings.Builder{}
```
All functions of the Builder use pointer receivers and this is consistent for all calls.

Reply:
Good point @wilfred-s , addressed it now.